e2e/cephfs_helper.go

```diff
@@ -98,6 +98,23 @@ func createCephfsSecret(f *framework.Framework, secretName, userName, userKey st
 	return err
 }
 
+func unmountCephFSVolume(f *framework.Framework, app *v1.Pod, pvc *v1.PersistentVolumeClaim) error {
+	app, err := f.ClientSet.CoreV1().Pods(f.UniqueName).Get(context.TODO(), app.Name, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	pv, err := f.ClientSet.CoreV1().PersistentVolumes().Get(context.TODO(), pvc.Spec.VolumeName, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	cmd := fmt.Sprintf("umount /var/lib/kubelet/pods/%s/volumes/kubernetes.io~csi/%s/mount", app.ObjectMeta.UID, pv.ObjectMeta.Name)
```

Comment:
The E2E should be as below

* Get list of running cephfs plugin pods
* Run through all cephfsplugin pods and run `mount |grep path`  and check application is mounted in cephfs plugin pod
* If the mount is present than umount it.